src/borg/remote.py

```diff
@@ -106,20 +170,29 @@ def serve(self):
                     return
                 unpacker.feed(data)
                 for unpacked in unpacker:
-                    if not (isinstance(unpacked, tuple) and len(unpacked) == 4):
+                    if isinstance(unpacked, dict):
+                        dictFormat = True
+                        msgid = unpacked[b"i"]
+                        method = unpacked[b"m"].decode('ascii')
+                        args = decodeKeys(unpacked[b"a"])
```

Comment:
ok, i see you're trying to save some bytes on the wire here by using short names. does it make a difference?